docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,679 @@
+
+- Core project:
+  - unstyled components and hooks, to push the customization level of components even further
+  - second design system, that will help us grow MUI beyond Material Design
+- X project:
+  - continue our focus on the `DataGrid` component, by releasing the stable 5.0 version and developing more features
+  - expand to a second component as part of the X package
+
+You can find more information about the new areas we are going to work on in the next sections.
+
+### Unstyled components and hooks
+
+To continue expanding on the customization effort, we plan to provide a version of the components that contain only the core functionality and accessibility support features, allowing you to add whatever styling strategy you wish. We're just starting and there's a lot of work to do to have a full set of unstyled components.
+
+You can read more about them in [the docs](/customization/unstyled-components/) and keep track of our progress in [#27170](https://github.com/mui-org/material-ui/issues/27170).
+
+Our high-level plan is to use the unstyled components and hooks as the basis of the Material components and second design system.
+We are aiming to complete this work with the next major release (v6).
+As of now, you can evaluate the unstyled primitives in the `@mui/base` package, or check out the next implementation of the Material Design components in the `@mui/material-next` package (targeted at v6).
+Please note that both packages are in an alpha state so that we can release breaking changes –
+we want to take the opportunity to create the best APIs we possibly can.
+
+You can help us shape these new packages by taking part in discussions.
+There are [RFCs](https://github.com/mui-org/material-ui/issues?q=is%3Aopen+label%3Adiscussion+%5BRFC%5D) waiting for responses.
+Don't hesitate to let us know what you think!
+
+### Second design system
+
+Developing a new design system is one of the main initiatives, alongside this new brand and the unstyled components, to grow MUI beyond Material Design. Its codename is **Joy** and we have just started [the high-level discussion](https://github.com/mui-org/material-ui/discussions/27803) around it.
+
+It will be built on top of our foundation packages (the base/unstyled components and `@mui/system`) to provide a close DX and API as Material Design. We plan on having built-in CSS variables generated from the theme, a great dark mode API, a custom look and feel, and more.
+
+We're prioritizing developing first the most frequently used components as we plan on releasing them continuously with each milestone. We intend on having on Joy all the components already available on Material Design so that developers can then pick between them focusing on the design they like the most. The main difference will be at the theme structure, where we want to achieve the right amount of flexibility while providing an amazing design by default.
```

Comment:
```suggestion
We're aiming for developing the most frequently used components first, as we plan on releasing them continuously with each milestone. We intend to have Joy versions of all components already available on Material Design so that developers can then pick between them, choosing the design they like the most. The main difference will be the theme structure, where we want to achieve the right amount of flexibility while providing an amazing design by default.
```